### ISSUE TITLE
Output of grayscale depthmaps

### ISSUE THREAD

**Issue by herrartist** [apple/ml-depth-pro#65]
How can I output grayscale depthmaps (and nothing else)?

**Comment by Dhruva-Storz** [comment]
Not sure if depth pro has an option for this, but the output of the model is a metric depth array. You need to normalize the depth values between 0 and 255 to get a traditional 8 bit greyscale image (and possibly invert it), then save the file as a PNG or whatever format.

**Comment by hradec** [comment]
You can run depth_pro and save a floating point tif file using something like this snippet:
```
        import depth_pro
        import torch, cv2

        def get_torch_device() -> torch.device:
            """Get the Torch device."""
            device = torch.device("cpu")
            if torch.cuda.is_available():
                if torch.cuda.get_device_properties(0).total_memory/1024/1024/1024 > 7:
                    device = torch.device("cuda:0")
            elif torch.backends.mps.is_available():
                device = torch.device("mps")
            return device

        # Load model and preprocessing transform
        model, transform = depth_pro.create_model_and_transforms(
            device=get_torch_device(),
            precision=torch.half,
        )
        self.progress_callback()
        model.eval()

        # Load and preprocess an image.
        image, _, f_px = depth_pro.load_rgb("your_rgb_image.jpg")
        image = transform(image)

        # Run inference.
        prediction = model.infer(image, f_px=f_px)
        depth = prediction["depth"].detach().cpu().numpy().squeeze()
        cv2.imwrite(f"{self.depthFile(self.frame)}_depthPro.tif", depth)
```
